admin/worker/reconcile_deployments.go

```diff
@@ -0,0 +1,127 @@
+package worker
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/rilldata/rill/admin"
+	"github.com/rilldata/rill/admin/database"
+	"github.com/rilldata/rill/runtime/pkg/observability"
+	"go.uber.org/zap"
+)
+
+const reconcileAllDeploymentsForProjectTimeout = 5 * time.Minute
+
+func (w *Worker) reconcileDeployments(ctx context.Context) error {
```

Comment:
Since "reconcile" means something else in the context of our runtime deployments, could we find another name for it here? Maybe "checkDeployments" or "validateDeployments" or something like that? If you want to stick with "reconcile", then add a docstring clarifying the difference.

Reply:
Agree, that's an undesirable naming collision, I've changed it to `validate_deployments`